test/models/solution_test.rb

```diff
@@ -30,4 +30,17 @@ class SolutionTest < ActiveSupport::TestCase
 
     refute solution.auto_approve?
   end
+
+  test "active_mentors" do
+    solution = create :solution
+    user1 = create :user
+    user2 = create :user
+
+    create :solution_mentorship, solution: solution
+    create :solution_mentorship, solution: solution, user: user1
+    create :track_mentorship, user: user1
+    create :track_mentorship, user: user2
+
+    assert_equal [user1], solution.active_mentors
```

Comment:
Can you clarify in the test name (or somehow), why user2 is not a mentor?

From what I'm reading:

Both of these users are mentors on the track, but only user1 has commented on this particular solution, therefore user2 would not be a mentor?